terraform/keycloak-sandbox/root_locals.tf

```diff
@@ -0,0 +1,16 @@
+locals {
+  environment  = var.environment
+  tag_prefix   = "test-keycloak"
+  aws_region   = "eu-west-2"
+  # The default VPC in the Sandbox environment
+  vpc_id       = "vpc-04f38e6c"
+  # Public subnets in the default VPC
+  subnet_ids    = ["subnet-7706850d", "subnet-a4d706e8"]
+  frontend_url = "https://example.com"
+  common_tags  = map(
+    "Environment", var.environment,
+    "Owner", "TDR",
+    "Terraform", true
+    # "CostCentre", data.aws_ssm_parameter.cost_centre.value
```

Comment:
Should we not be including the cost centre, even though the resources should be destroyed after use?

Reply:
Oops, I meant to update this. Thanks for spotting it!

I've manually added a cost centre SSM parameter and referenced it here. It's a little bit fragile, in that we don't have anything that automatically sets up parameters in the sandbox environment, but hopefully it's OK because it's just for a development script.